Modules/+Sources/+msquared/EMM.m

```diff
@@ -147,6 +147,15 @@ function tune(obj,target)
             obj.etalon_lock = true;  % We don't know at this point anything about etalon if not locked
             obj.updatingVal = false;
         end
+        %temp modification to get emm wavelength
+        function [wavelength] = getWavelength(obj)
+            try
+                response = obj.emmHandle.hwserver.com('wavemeter','GetSummary');
+                wavelength = response([response.channel] == 7).wavelength;
```

Comment:
This is hardcoded as `7` for a reason? Should at the very least grab the emm's wavemeter channel I'd think